stable/newrelic-infrastructure/templates/daemonset.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if and .Values.licenseKey .Values.cluster }}
+{{- if and (or .Values.licenseKey .Values.customSecret) .Values.cluster }}
```

Comment:
Does this work ok if I set `.Values.custom.Secret.name` ? I've not seen this syntax before, just interested ;)

Reply:
It works for either `.Values.licenseKey` or `.Values.customSecret` being set. But theoretically should not evaluate `.Values.customSecret` if `.Values.licenseKey` is set (that is common in programming languages but I have not tested that explicitly)

Reply:
Checked and yep it works, nice 👍

Reply:
Could we extract this to a helper instead? I'm not very easy to understand what happens here.

Off topic: this reminds me old times of emulating `if` inside Python's list comprehensions.

Reply:
I think off-loading computation into function which will be used once will make it even less readable. And I think that is written in Polish notation, commonly used in functional languages like lisp, can we leave it as is ?

Reply:
Personally, I don't mind. I just want to be friendly with other people that might contribute and are not big fans of polish notation, hahaha. This can be improved later on tough, not a requirement for the PR.